do-catalog/src/views/DatasetData.vue

```diff
@@ -225,22 +186,27 @@ export default {
     showTooltip(variableName, event) {
       let tooltipInfo = this.findVariableInfo(variableName);
       let tableBoundingSize = this.$refs.tableWrapper.getBoundingClientRect();
-      let targetBoundingSize = event.target.getBoundingClientRect();
-      this.tooltip.left = targetBoundingSize.left - tableBoundingSize.left;
-      if (this.tooltip.left < 60) {
+      // let targetBoundingSize = event.target.getBoundingClientRect();
+      // this.tooltip.left = targetBoundingSize.left - tableBoundingSize.left;
+      this.tooltip.left = event.clientX - 22;
+      if (this.tooltip.left < 140) {
         this.tooltip.isFirst = true;
+        this.tooltip.left -= 26
       } else if (tableBoundingSize.width - this.tooltip.left < 120) {
         this.tooltip.isLast = true;
-        this.tooltip.left += targetBoundingSize.width;
+        // this.tooltip.left += targetBoundingSize.width;
+        this.tooltip.left += 26
       } else {
-        this.tooltip.left += targetBoundingSize.width / 2;
+        this.tooltip.isFirst = false;
+        this.tooltip.isLast = false;
+        // this.tooltip.left += targetBoundingSize.width / 2;
       }
 
-      if (this.tooltip.left < -20) {
-        this.tooltip.left = -20;
-      } else if (tableBoundingSize.width - this.tooltip.left < -20) {
-        this.tooltip.left = tableBoundingSize.width + 20;
-      }
+      // if (this.tooltip.left < -20) {
+      //   this.tooltip.left = -20;
+      // } else if (tableBoundingSize.width - this.tooltip.left < -20) {
+      //   this.tooltip.left = tableBoundingSize.width + 20;
```

Comment:
Good simplification! I have to test it in my browsers but it looks good!
BTW, remove all unneeded code to show its real beauty :wink:

Reply:
True that. :fire: